.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on:
+  push:
+    branches: [master, actions]
```

Comment:
I think we can skip the `actions` branch here;  that was just something I used when developing the action for indico and then forgot to remove